scripts/python/packages/parthenon_tools/parthenon_tools/phdf_diff.py

```diff
@@ -211,14 +211,17 @@ def compare_metadata(f0, f1, quiet=False, one=False, check_input=False, tol=1.0e
         if one:
             return ret_code
 
-    # Compare the names of attributes in /Info, except "Time"
+    # Compare the names of attributes in /Info, except those we know
+    # may vary safely
     f0_Info = {
         key: value
         for key, value in f0.Info.items()
         if key != "Time"
         and key != "BlocksPerPE"
         and key != "WallTime"
         and key != "OutputFormatVersion"
+        and key != "BoundaryConditions"
+        and key != "SwarmBoundaryConditions"
```

Comment:
This is safe to exclude from metadata checks becuase it was always unused by the infrastructure and this MR (dramatically) changes what this output looks like.